manifests/download.pp

```diff
@@ -0,0 +1,64 @@
+# == Definition: archive::download
```

Comment:
it wasn't documented in camptocamp/archive either ~_~
i don't *want* to document it, because it's just a shim. private api that moves as camptocamp/archive moves.